guides/common/modules/proc_installing-the-dns-infoblox-module.adoc

```diff
@@ -15,11 +13,12 @@ DNS records are managed only in the default DNS view, it's not possible to speci
 --foreman-proxy-dns true \
 --foreman-proxy-dns-provider infoblox \
 --foreman-proxy-plugin-dns-infoblox-dns-server _infoblox.example.com_ \
---foreman-proxy-plugin-dns-infoblox-dns-view default \
+--foreman-proxy-plugin-dns-infoblox-dns-view _default_ \
 --foreman-proxy-plugin-dns-infoblox-password infoblox \
 --foreman-proxy-plugin-dns-infoblox-username admin
 ----
 +
-Optionally, you can change the value of the `--foreman-proxy-plugin-dns-infoblox-dns-view` option to specify a DNS Infoblox view other than the default view.
-. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}* and select the {SmartProxy} with the Infoblox DNS module and click *Refresh*.
-. Ensure that the *dns* features are listed.
+Optionally, you can change the value of the `--foreman-proxy-plugin-dns-infoblox-dns-view` option to specify an Infoblox DNS view other than the default view.
+. Optional: In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*, select the {SmartProxy} with the Infoblox DNS module, and ensure that the *dns* feature is listed.
+. In the {ProjectWebUI}, navigate to *Infrastructure* > *Domains*.
+. For all domains managed through Infoblox, ensure that the DNS {SmartProxy} and Reverse DNS {SmartProxy} are set for those domains.
```

Comment:
Not sure how I missed it previously, but reverse DNS is set on the subnet and not on a domain.

Reply:
Originally, it was in the DHCP

Reply:
The DNS module